xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVTB.cpp

```diff
@@ -160,6 +183,11 @@ bool CRendererVTB::RenderHook(int index)
   return true;
 }
 
+void CRendererVTB::AddSupportedHwRenderFormats()
```

Comment:
Mhhh are you sure? m_formats is used in "GetRenderInfo" for example. If the renderer doesn't add that format to it - who does instead? What you stated would mean i could remove m_formats completely - but i don't see how.